.devcontainer/Dockerfile

```
@@ -24,8 +24,7 @@ COPY . /app
 RUN --mount=type=cache,target=/root/.cache/uv \
     --mount=type=bind,source=uv.lock,target=uv.lock \
     --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
-    uv venv && uv sync --group dev --group tests \
-        --extra duckdb --extra clickhouse  --extra examples --extra geospatial
+    uv venv && just sync-light
```

If you take my suggestion from below this becomes:

```suggestion
    uv venv && just sync duckdb
```